File: app/src/main/java/bzh/naoned/naonedbusdatadisplay/service/model/Target.java
package bzh.naoned.naonedbusdatadisplay.service.model;

import android.graphics.Color;

import com.google.gson.annotations.SerializedName;

import io.realm.RealmObject;

public class Target extends RealmObject {
    @SerializedName("routeCode")
    private String mRouteCode;
    @SerializedName("routeLabel")
    private String mRouteLabel;
    @SerializedName("routeColor")
    private String mRouteColor;

    public String getRouteCode() {
        return mRouteCode;
    }

    public String getRouteLabel() {
        return mRouteLabel;
    }

    public int getRouteColor() {
        return Color.parseColor(mRouteColor);
    }
}

File: app/src/main/java/bzh/naoned/naonedbusdatadisplay/service/model/Target.kt
package bzh.naoned.naonedbusdatadisplay.service.model

import android.graphics.Color

import com.google.gson.annotations.SerializedName

import io.realm.RealmObject

open class Target : RealmObject() {
    @SerializedName("routeCode")
    var routeCode: String? = null
    @SerializedName("routeLabel")
    var routeLabel: String? = null
    @SerializedName("routeColor")
    private var mRouteColor: String? = null

    fun getRouteColor(): Int? {
        return Color.parseColor(mRouteColor)
    }
}
